content/en/docs/tasks/security/authentication/claim-to-header/index.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Copy Jwt Claim to HTTP Header
```

Comment:
We seem to use JWT elsewhere. Should this be uppercased as well as in the description (and maybe elsewhere)?